c2cgeoportal/scaffolds/create/docker-run_tmpl

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python
+
+import sys
+import os
+import subprocess
+import getpass
+
+
+def main():
+
+    args = sys.argv[1:]
+    docker_cmd = ['docker', 'run']
+
+    root = False
+    if args[0] == '--root':
+        args = args[1:]
+        root = True
+
+    if args[0][0] == '-':
+        docker_cmd.append(args[0])
+        args = args[1:]
+
+    # docker_adrs = netifaces.ifaddresses('docker0')[2][0]['addr']
+
+    docker_cmd.extend([
+        "--rm",
+        "--volume=/var/run/docker.sock:/var/run/docker.sock",
+        "--volume=build:/build",
+        "--volume={pwd}:/src".format(pwd=os.getcwd()),
+        "--volume={home}:{home}".format(home=os.environ["HOME"]),
+        "--env=USER_NAME={}".format(getpass.getuser()),
+        "--env=USER_ID={}".format(os.getuid()),
+        "--env=GROUP_ID={}".format(os.getgid()),
+    ])
+
+    for env in [
+        "HOME",
+        "TRAVIS",
+        "TRAVIS_BRANCH",
+        "TRAVIS_JOB_ID",
+        "SRID",
+        "EXTENT",
+    ]:
+        docker_cmd.append("--env={}={}".format(env, os.environ.get(env, "")))
+
+    if root:
+        docker_cmd.extend(["--entrypoint", args[0]])
+        args = args[1:]
+
+    docker_cmd.append("camptocamp/geomapfish_build:{{geomapfish_version}}")
```

Comment:
Shouldn't this script be in update?

Reply:
I will change the upgrade procedure ant this will be easy to be upgraded from the create template.